Resources/Prototypes/Entities/Structures/Furniture/Tables/operating_table.yml

```
@@ -13,8 +13,18 @@
   # Begin Shitmed Changes
+  - type: Construction
+    graph: Machine
+    node: machine
+    containers:
+     - machine_board
+     - machine_parts
   - type: Machine
     board: OperatingTableCircuitboard
+  - type: ContainerContainer
+    containers:
+      machine_board: !type:Container
```

yamllint (1.35.1)

[error] wrong indentation: expected 5 but found 6

(indentation)